books_project/books_app/components/author_component.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime
+from books_app.repositories.author_repository import AuthorRepository
+from rest_framework.exceptions import PermissionDenied, NotFound
+
+class AuthorComponent:
+
+    def create_author(self, name , birth_date, nationality):
+        author_birth_date = datetime.strptime(birth_date, '%Y-%m-%d').date()  # Converts string to date object
+        AuthorRepository.create_author(name,author_birth_date,nationality)
```

Comment:
Please use named parameters, so in this case: 
AuthorRepository.create_author(name=name, author_birth_date=author_birth_date, nationality=nationality)
And leave a space after every comma, following python conventions